docs/user-guide/deploy-on-OSP.md

```diff
@@ -1,2 +1,69 @@
-# Openshift deployment with OAS - On Openstack
-work in progress...
+# Openshift Deployment on OpenStack by OpenShift Assisted Service 
+
+This guide explains how to deploy OpenShift by the OpenShift Assisted Service
+on OpenStack.
+
+---
+**NOTE**
+
+Currently, the deployment of an OpenShift Cluster on
+Red Hat OpenStack Platform is blocked by OpenShift Assisted Service, while RDO is working fine.
+The related check can be disabled by including `valid-platform` the environment variable
+`DISABLED_HOST_VALIDATIONS` in the context of the OpenShift Assisted Service, e.g. like this:
+```
+DISABLED_HOST_VALIDATIONS=valid-platform,container-images-available 
+```
+
+---
+
+## Requirements
+
+* Two floating IP addresses, it is recommended that DNS resolves
+  * api.CLUSTERNAME.DOMAIN to the first floating IP address, and
+  * oauth-openshift.apps.CLUSTERNAME.DOMAIN and
+    console-openshift-console.apps.CLUSTERNAME.DOMAIN to the second one
```

Comment:
you actually need a wildcard, it's well documented here: https://github.com/openshift/installer/tree/master/docs/user/openstack#create-api-and-ingress-dns-records

Reply:
Thanks a lot!